inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sources/reader/KafkaReader.java

```diff
@@ -143,7 +148,9 @@ public Message read() {
             }
         } else {
             // commit offset
-            consumer.commitAsync();
+            if (!isSourceExist()) {
```

Comment:
when source not exist commit offset? should commit offset when source exist

Reply:
you shou see the method isSourceExist()

Reply:
isSourceExist() return false indicates that  source does not exist